spec/src/main/asciidoc/ch08-entity-packaging.adoc

```diff
@@ -78,6 +81,21 @@ of a Java SE application, it is necessary to explicitly list the managed
 persistence classes included in the persistence unit using the `class`
 element of the `persistence.xml` file. See <<a12305>>.
 
+==== Persistence Properties Files
```

Comment:
Another question: To use the properties files, do I need to use the new PersistenceModule annotation? Or I can define a properties file also for PUs defined in persistence.xml?

Since the properties file name is tied with PU name, it should also make sense to use it with any PU, not only a PU in a persistence module, right? If yes, I think it makes sense to split this proposal about properties files into a separate PR.